README.md

```diff
@@ -42,8 +42,15 @@ This part describes the Docker image that this repo is designed to build. People
 - `MW_RECAPTCHA_SITEKEY`
 - `MW_RECAPTCHA_SECRETKEY`
 - `PLATFORM_API_BACKEND_HOST`: points to an internal mode wbstack api service
-- `MW_ELASTICSEARCH_HOST`: elasticsearch hostname
-- `MW_ELASTICSEARCH_PORT`: elasticsearch port
+- `MW_ELASTICSEARCH_HOST`: DEPRECATED: elasticsearch hostname. If set "default" settings below are ignored.
```

Comment:
Question about marking this as `DEPRECATED`. Considering we have sailed through a super smooth migration and now want to be using ES7 only. Would we not want to go back to using this setting?

Reply:
I was thinking that we would want to leave the capability to have a second write only cluster there.
For example, if we want to migrate to a second 7 cluster in the future with different field mappings